_data/hosting_providers.json

```diff
@@ -2379,4 +2379,14 @@
     "reviewed": "2020.04.01",
     "note": "Available for all membership types, without any limitations."
   },
+    {
+    "name": "HahuCloud",
+    "link": "https://www.hahucloud.com",
+    "category": "partial",
+    "tutorial": "",
```

Comment:
Do you have a page that explains the steps users need to take to turn on SSL certificates on your service? If so, please a link here.

Reply:
At the moment we don't have a tutorial for that.